# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/moz.build
# Commit: ae0715e32c27
# Full Hash: ae0715e32c271e832a028e7e90d66b63e1e07c35
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Update AudioConduit to use new APIs. r=ng
#   
#   This also disables the tests while we figure out the new API.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130064
# ==============================================================================

diff -r 26f3fec64ff0 -r ae0715e32c27 media/webrtc/signaling/gtest/moz.build
--- a/media/webrtc/signaling/gtest/moz.build	Fri Oct 30 11:39:15 2020 -0400
+++ b/media/webrtc/signaling/gtest/moz.build	Fri Oct 30 14:22:28 2020 -0400
@@ -39,7 +39,7 @@
     ]
 
     SOURCES += [
-        "audioconduit_unittests.cpp",
+#        "audioconduit_unittests.cpp",
         "jsep_session_unittest.cpp",
         "jsep_track_unittest.cpp",
         "mediapipeline_unittest.cpp",
